price/priceService.ts

```diff
@@ -1,9 +1,23 @@
 import { baseUrl } from './url.js';
+import {Pair} from '../types/common.js'
 
-type Pair = string;
 type TimeFrame = string;
 
-async function fetchData(pair: Pair, timeFrame: TimeFrame): Promise<string> {
+interface candles {
+  startedAt: "string",
+  ticker: "string",
+  resolution: "1MIN",
+  low: "string",
```

Comment:
Are you sure about these types? It makes sense for these to be `number`.

Reply:
yes they are string type when I look at the response they be a string and in dydx site they be string too.
but I think they can be number too . I add the number type via union type